src/random.jl

```diff
@@ -110,3 +112,43 @@ function whoisproducer(A)
     isproducer = any(A, dims = 2)
     return isproducer
 end
+
+"""
+**Create facilitation links**
+
+Takes trophic matrix (A) and number of facilitation links wanted (L). 
+Returns matrix of facilitation links. 
+Facilitating species are in rows, facilitated species are in columns.
+"""
+function facilitationlinks(A, L)
+    S = size(A,1) # number of species 
```

Comment:
I think julia autoformatter would have written `size(A, 1)`. Think of future diffs/merge conflicts and watch your formatting ;)